### ISSUE TITLE
Will XLForm officially support multi-level selector?

### ISSUE THREAD

**Issue by yisheng** [xmartlabs/XLForm#394]
Like city selector (Country -> State -> Region).

**Comment by littlehaker** [comment]
+1

**Comment by mtnbarreto** [comment]
@yisheng  
Currently XLForm doesn't have support for this kind of selector. 
Have you created a custom cell to get this behaviour?
Regards

**Comment by yisheng** [comment]
@mtnbarreto 

Yes, I created a custom form cell & a selector view controller to support it. But the codes are not optimized for general use, maybe I can try to open source it.

**Comment by techistoner** [comment]
@yisheng  can you open source  or post a url ?
